cmp/fc58b79f-8fbf-45d3-b97c-2faaa02279f6/component.lp

```diff
@@ -0,0 +1,117 @@
+(librepcb_component fc58b79f-8fbf-45d3-b97c-2faaa02279f6
+ (name "FTDI FT232R")
+ (description "FT232R USB UART")
+ (keywords "ftdi,rs232,usb")
+ (author "Martijn Braam")
+ (version "0.1")
+ (created 2018-12-02T22:19:13Z)
+ (deprecated false)
+ (category 58601391-795b-4ebf-ae55-984352167ce4)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER}}")
+ (prefix "IC")
```

Comment:
For ICs we use the prefix "U":

```suggestion
 (prefix "U")
```

Reply:
I'll update it

I used this because that's what used in the opamp example so I guess the docs need an update: https://docs.librepcb.org/#gettingstarted-libraries-sym

Reply:
Good point, I created an issue in the docs repo: https://github.com/LibrePCB/librepcb-doc/issues/27